cylc/flow/main_loop/__init__.py

```diff
@@ -43,7 +43,7 @@
 
   .. code-block:: console
 
-     $ # run a workflow using the "health check" and "auto restart" plugins:
+     $ # run a workflow using the "health check" and "auto restart" plugins:
```

Comment:
What caused these - some weird config in your code editor @oliver-sanders ?

Reply:
alt+space on Mac OS - I have a pre-commit hook now (FYI there is a commit to remove them on the /usr/bin/env branch, it just hasn't been merged yet).